lib/components/canvas/_canvas_painter.dart

```diff
@@ -56,7 +56,7 @@ class CanvasPainter extends CustomPainter {
           needToRestoreCanvasLayer = true;
         }
         if (currentSelection?.strokes.contains(stroke) ?? false) {
-          paint.color = primaryColor;
+          paint.color = Color.lerp(stroke.strokeProperties.color.withInversion(invert), Colors.black, 0.25) ?? primaryColor;
```

Comment:
Why did you choose `Colors.black` over `primaryColor`?

Reply:
I chose black so that the color is only darkened and does not appear distorted due to the primaryColor.
With the primaryColor the balance between selection feedback and color change feedback is not given.

Here are some example pictures:
https://imgur.com/a/t5IBtnO

Notice how you cant really see that "Test" was selected sucessfully.
But with the current implementation you have the same effect with black text.

Reply:
> But with the current implementation you have no selection feedback on black text.

That was my concern. I think black text should take priority over coloured text since most people write in black.

Please can you change `Colors.black` to `primaryColor`?

Reply:
This is how it looks now:
![Screenshot_2023-09-21-21-15-08-232_com adilhanney saber](https://github.com/adil192/saber/assets/40892690/ed6ebb8d-03eb-42d3-9fe7-39613b2ef61e)
![Screenshot_2023-09-21-21-15-16-646_com adilhanney saber](https://github.com/adil192/saber/assets/40892690/f7c2f87b-9951-4914-ad45-bfc4846cb508)
![Screenshot_2023-09-21-21-18-48-146_com adilhanney saber](https://github.com/adil192/saber/assets/40892690/8c2e8f5e-d2d8-4f0a-abf8-1104b9b844ea)
![Screenshot_2023-09-21-21-19-02-960_com adilhanney saber](https://github.com/adil192/saber/assets/40892690/444d8532-9323-4a8d-a9a6-0d2cd3e225b2)